src/Faithlife.Data/DbConnectorCommand.cs

```diff
@@ -35,6 +35,11 @@ namespace Faithlife.Data
 		/// </summary>
 		public bool IsCached { get; }
 
+		/// <summary>
+		/// True after <see cref="Prepare"/> is called.
+		/// </summary>
+		public bool IsPrepared { get; }
```

Comment:
#12 called for this, but I'm unsure of the value of this property.

Some databases _do_ expose `XyzCommand.IsPrepared` that reflects the status of the underlying command, e.g., https://github.com/mysql-net/MySqlConnector/blob/06cabf777328b7aaee4a94987b61953d3d3ba99b/src/MySqlConnector/MySqlCommand.cs#L188

Others implement the `Prepare` method but it's a no-op (see https://github.com/Faithlife/FaithlifeData/pull/17#issuecomment-729871754).

Since this property (in the general case) will only actually mean `HasDbCommandPrepareBeenCalled`, I'm not sure it's valuable to add.

Reply:
It could (as could `IsCached`) be a private field, but I see no harm in exposing it, e.g. for debugging purposes.